vm/actor/src/lib.rs

```diff
@@ -53,6 +53,16 @@ where
     Map::<_, V>::new_with_bit_width(store, HAMT_BIT_WIDTH)
 }
 
+/// Create a hamt with a custom bitwidth.
+#[inline]
+pub fn make_map_with_bitwidth<BS, V>(store: &'_ BS, bitwidth: u32) -> Map<'_, BS, V>
+where
+    BS: BlockStore,
+    V: DeserializeOwned + Serialize,
+{
+    Map::<_, V>::new_with_bit_width(store, bitwidth)
+}
```

Comment:
Optional: you can have duplicates where the original uses the default hamt bit width, or you could just pass in the width for each. I'd argue it would be less prone to an error if you did the latter, so this wouldn't implicitly fail if someone didn't know about the default. Very opinionated though and this could definitely be better, but having 4 functions with different names for all combinations may be confusing

Reply:
Yeah, youre right. Best thing is to mandate that a bitwidth be passed through